src/OrchardCore/OrchardCore.Localization.Core/OrchardCore.Localization.Core.csproj

```diff
@@ -13,6 +13,10 @@
     <FrameworkReference Include="Microsoft.AspNetCore.App" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="System.Linq.Async" />
+  </ItemGroup>
```

Comment:
This doesn't seem necessary.

Reply:
It's necessary to convert from `IAsyncEnumerable` to `Enumberable`

Reply:
It still builds if I remove it.

Reply:
Really!! I got 4 errors, how the `IAsyncEnumerable` extension methods be there without referencing a proper package?

Reply:
I dunno :).